pkg/network/protocols/postgres/model_linux.go

```diff
@@ -26,6 +27,7 @@ type EventWrapper struct {
 	operation    Operation
 	tableNameSet bool
 	tableName    string
+	normalizer   *sqllexer.Normalizer
```

Comment:
who generates the normalizer?

I think you should introduce `NewEventWrapper` method to ensure the normalizer is being generated, otherwise, we can reach into panic

Reply:
Added